roshambo-ai-model/kustomize/bases/inference/serving-runtime.yaml

```diff
@@ -0,0 +1,55 @@
+﻿---
+apiVersion: serving.kserve.io/v1alpha1
+kind: ServingRuntime
+metadata:
+  name: model-server
+  labels:
+    name: model-server
+    opendatahub.io/dashboard: 'true'
+  annotations:
+    enable-auth: 'false'
+    enable-route: 'false'
+    opendatahub.io/disable-gpu: 'true'
+    opendatahub.io/template-display-name: OpenVINO Model Server
+    opendatahub.io/template-name: ovms
+    openshift.io/display-name: Model Server
+spec:
+  builtInAdapter:
+    memBufferBytes: 134217728
+    modelLoadingTimeoutMillis: 90000
+    runtimeManagementPort: 8888
+    serverType: ovms
+  containers:
+    - args:
+        - '--port=8001'
+        - '--rest_port=8888'
+        - '--config_path=/models/model_config_list.json'
+        - '--file_system_poll_wait_seconds=0'
+        - '--grpc_bind_address=127.0.0.1'
+        - '--rest_bind_address=127.0.0.1'
+      image: >-
+        quay.io/opendatahub/openvino_model_server@sha256:20dbfbaf53d1afbd47c612d953984238cb0e207972ed544a5ea662c2404f276d
+      name: ovms
+      resources:
+        limits:
+          cpu: '2'
+          memory: 8Gi
+        requests:
+          cpu: '1'
+          memory: 2Gi
```

Comment:
I assume this is the lowest safe limit? Seems high, but probably is what it is and is needed?